README.md

```diff
@@ -1,50 +1,34 @@
-[![Stories Backlog](https://badge.waffle.io/Wycliffe-USA/rapid-translation-toolkit.png?label=Backlog&title=Backlog)](http://waffle.io/Wycliffe-USA/rapid-translation-toolkit)
+[![Stories in Ready](https://badge.waffle.io/Wycliffe-USA/rtt-linguiflow.png?label=ready&title=Ready)](http://waffle.io/Wycliffe-USA/rapid-translation-toolkit)
+[![Stories In Progress](https://badge.waffle.io/Wycliffe-USA/rtt-linguiflow.png?label=in%20progress&title=In%20Progress)](http://waffle.io/Wycliffe-USA/rapid-translation-toolkit)
 
-# Wycliffe@Urbana15: #Hacktranslation
+# Linguiflow
 
-We aim to bring the power of natural language processing (NLP) to bear on the work of Bible translation.
+#### A Rapid Translation Toolkit project
 
-## The problem
+---------
 
-Bible translation is a task that takes many years of dedicated, hard work from a large number of people. One of the major tasks, accounting for a large portion of the time required in translation, is gathering properly defined vocabulary and identifying parts of speech. An existing process called [Rapid Word Collection](http://rapidwords.net/) does an excellent job of collecting a large number of words in a very short period of time. However, the process is almost entirely manual and dependent on human intervention. We would like to develop processes leveraging NLP tools (or even other things. We aren't picky!) that would allow us to 'discover' more words in the target language in a shorter amount of time, using fewer resources.
+### Introduction
 
-Many of these languages are poor in written literature, so large sets of words are hard to come by. Some are still largely unwritten. Almost all are in areas where internet connectivity is scarce, slow, or unreliable. The translation activity is happening in remote areas where power is also unreliable and shipping equipment to the site is a gamble.
+Linguiflow is aimed at assisting translators and linguists in the field working with low-resources languages. When first learning and documenting a language for translation purposes, one of the major hurdles is simply gathering, learning, and cataloging words. Many great tools already exist for data organization and cataloging, such as [the Field Linguist's Toolbox](http://www-01.sil.org/computing/toolbox/) and methods for collecting words, such as [Rapid Word Collection](http://rapidwords.net/), both projects from [SIL](http://www.sil.org/). 
 
-## Vagrant Setup Instructions
+Linguiflow aims to further assist and augment field translators in the word discovery process by taking parallel text in two languages, one well-known and one low-resource language, and analyze them to try and determine word pairs between the two. This could provide the translator with additional confidence in defining a word, help the translator discover latent phrases, or provide additional suggestions and alternative words that are used in similar ways.
```

Comment:
"and analyze" is in the wrong tense.
"and analyzing" would fix that.
That's my only gripe :)

(This sentence is very wordy; Maybe split into multiple sentences in the future?)